Gemfile

```diff
@@ -9,7 +9,7 @@ group :develop do
   gem "chef", "~> 12.9"
   gem "chefspec", "~> 4.5"
   gem "emeril", "~> 0.8"
-  gem "librarian-chef"
+  gem "berkshelf", "~> 5.0"
```

Comment:
Be aware this won't install within the current ChefDK since it's Ruby 2.1.9 and this requires Ruby 2.2+